libs/utils/federated.js

```diff
@@ -20,10 +20,7 @@ export const getFederatedContentRoot = () => {
     : 'https://www.adobe.com';
 
   if (origin.includes('localhost') || origin.includes('.hlx.')) {
-    // Akamai as proxy to avoid 401s, given AEM-EDS MS auth cross project limitations
-    federatedContentRoot = origin.includes('.hlx.live')
-      ? 'https://main--federal--adobecom.hlx.live'
-      : 'https://www.stage.adobe.com';
+    federatedContentRoot = `https://main--federal--adobecom.hlx.${origin.includes('hlx.live') ? 'live' : 'page'}`;
```

Comment:
Nit to safely ignore: conditional on L22 and previous logic check for `.hlx.live` (dot prefix). This would probably not cause any errors, though. Since there's already a check for either `localhost` or `.hlx.`, you could also just check `origin.endsWith('.live')`